content/docs/06-integrations/00-ubuntu.md

```diff
@@ -125,7 +126,7 @@ Spectro Golden images include most of the hardening standards recommended by CIS
 
 # Ubuntu Advantage
 
-Canonical Ubuntu Advantage extends your infrastructure's security, certified compliance, and 24x7 support. With Palette, enable the UA services when modifying the Pack Values. See below for steps on how to modify the Preset options.
+Ubuntu Advantage services extend your infrastructure's security, compliance, and productivity requirements. With Palette, enable the UA services when modifying the Pack Values. See below for steps on how to modify the Preset options.

```

Comment:
Is there better way to represent "With Palette, enable the UA services when modifying the Pack Values."